src/mlpack/methods/decision_tree/all_categorical_split.hpp

```diff
@@ -62,7 +62,8 @@ class AllCategoricalSplit
       const WeightVecType& weights,
       const size_t minimumLeafSize,
       arma::Col<typename VecType::elem_type>& classProbabilities,
-      AuxiliarySplitInfo<typename VecType::elem_type>& aux);
+      AuxiliarySplitInfo<typename VecType::elem_type>& aux,
+      const double minimumGainSplit);
```

Comment:
This is also a bit picky, but we typically order const (input) parameters before output parameters, so I would put `minimumGainSplit` before `classProbabilities` (and similar changes through the rest of the code).